### PR TITLE
Python 3 support

### PR DESCRIPTION
**Pull request by janwijbrand**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
 
 
-version = '1.3.dev0'
+version = '2.0.dev0'
 description = "Importchecker finds unused imports in Python modules."
-test_requirements = ['mock', 'zope.testrunner']
+test_requirements = ['zope.testrunner']
+
+if sys.version_info.major == 2:
+    test_requirements.append('mock')
```
Comment (mgedmin) [inline setup.py]: This sort of dynamic dependency computation used to cause problems with certain versions of pip due to wheel caching.

The modern alternative is to use environment markers: https://hynek.me/articles/conditional-python-dependencies/

File: setup.py [hunk 2]
```diff
@@ -39,8 +43,11 @@ def read(*rnames):
         'Natural Language :: English',
         'Operating System :: OS Independent',
         'Programming Language :: Python',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.3',
```
Comment (mgedmin) [inline setup.py]: FWIW other zopefoundation projects are dropping Python 3.3 support.

Also, you're not running the tests on 3.3 (neither from Travis, nor from tox.ini), so I don't think it's right to claim it as a supported version in setup.py.

File: src/importchecker/importchecker.py [hunk 3]
```diff
@@ -17,39 +17,31 @@
...
+import ast
 import os
 import sys
 
 
 def _findDottedNamesHelper(node, result):
-    more_node = node
     name = node.__class__.__name__
-    if name == 'Getattr':
+    childs = ast.iter_child_nodes(node)
```
Comment (mgedmin) [inline src/importchecker/importchecker.py]: The usual English spelling is `children` ;)

**Review by mgedmin — APPROVED** [review]
LGTM, with minor comments

**Comment by janwijbrand** [comment]
I need to study the environment-markers-way of declaring conditional dependencies more to know how to effectively fix it for importchecker. If it causes problems in real-life I can come back to this and try to fix it.

**Pull request merged**